code/game/gamemodes/gameticker.dm

```diff
@@ -188,6 +188,13 @@ var/datum/controller/gameticker/ticker
 	collect_minds()
 	equip_characters()
 	current_state = GAME_STATE_PLAYING
+	//Handle all the cyborg syncing
+	for(var/mob/living/silicon/robot/R in player_list)
```

Comment:
mommis are robots too, did you test with them?

Reply:
No, I'll make an exception for them

Reply:
also there's a global list with all cyborgs in the world, use that one instead

Reply:
what is it called?

Reply:
my goof, there's none (only ais)